subprojects/core/src/integTest/groovy/org/gradle/internal/scan/config/BuildScanConfigIntegrationTest.groovy

```diff
@@ -36,9 +37,14 @@ Please see https://gradle.com/scans/help/gradle-incompatible-plugin-version for
                     def c = services.get(${BuildScanConfigProvider.name}).collect([getVersion: { "$pluginVersionNumber" }] as $BuildScanPluginMetadata.name) 
                     println "buildScan.enabled: " + c.enabled 
                     println "buildScan.disabled: " + c.disabled 
-                """
+                """.stripIndent()
```

Comment:
Why is adding `stripIndent()` useful? It looks like unnecessary noise to me.

Reply:
I removed it. I actually like adding `stripIdent` when I write integration tests, since then generated build files look nicer and sometimes I look at them. You prefer not using `stripIdent()` at all in integration tests?

Reply:
I tend to look far more frequently at the code of the integration test than at the actual build file. The difference in the build file is also very small: when you don't use `stripIndent()` the lines start a little to the right, so it's not even harder to read or anything. Therefore I think we should prefer cleaner integration tests over cleaner build files, and not to use `stripIndent()` without a stronger reason.